bool activate_manual_control

